hudi-flink-datasource/hudi-flink/src/test/java/org/apache/hudi/table/ITTestHoodieDataSource.java

```diff
@@ -1753,4 +1756,27 @@ private List<Row> execSelectSql(TableEnvironment tEnv, String select, String sin
         .flatMap(Collection::stream)
         .collect(Collectors.toList());
   }
+
+  private Row row(Object... args) {
+    return Row.of(args);
+  }
```

Comment:
Can we move these utilities to `TestData.java` ?

Reply:
Moved to `TestData`